Update text-moderation-api.md

relugas opened the pull request:
Added a link to the Text - Detect Language API reference page as this helped me understand how you can first pass in a string to obtain the language and then pass this into the screen text API, ensuring you are screening text against the correct language.

azurecla commented:
Hi __@gangsir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

relugas committed:
Update text-moderation-api.md

Added a link to the Text - Detect Language API reference page as this helped me understand how you can first pass in a string to obtain the language and then pass this into the screen text API, ensuring you are screening text against the correct language.

GitHubber17 committed:
Update text-moderation-api.md
